pkg/config/user_config.go

```diff
@@ -669,6 +673,7 @@ func GetDefaultConfig() *UserConfig {
 				UnstagedChangesColor:       []string{"red"},
 				DefaultFgColor:             []string{"default"},
 			},
+			CommitAuthorFormat:        "full",
```

Comment:
This should be set to `auto` if `auto` is the default value.

Reply:
fixed